ansible_runner/runner.py

```diff
@@ -337,7 +337,7 @@ def _decode(x):
                 if self.config.idle_timeout and (time.time() - self.last_stdout_update) > self.config.idle_timeout:
                     self.kill_container()
                     Runner.handle_termination(child.pid, is_cancel=False)
-                    self.timed_out = True
+                    self.idle_timed_out = True
```

Comment:
This will need a default like `timed_out` has on line 37. Otherwise if we _don't_ hit this case, we'll crash when we check the variable below.